projects/1.20-fabric/assets/observable/observable/lang/zh_cn.json

```
@@ -0,0 +1,34 @@
+{
+  "screen.observable.profile": "轮廓",
+  "screen.observable.results": "统计",
+  "key.observable.profile": "显示",
+  "category.observable.keybinds": "Observable",
+  "text.observable.profile_tps": "TPS 占用显示",
+  "text.observable.profile_fps": "FPS 占用显示",
+  "text.observable.sampler": "加载采样器的配置文件（增加性能消耗）",
+  "text.observable.unimplemented": "未完成！",
+  "text.observable.unavailable": "服务器上未安装Observable",
+  "text.observable.no_permissions": "您没有权限运行",
+  "text.observable.error": "执行操作时出错： %s",
+  "text.observable.report": "请在github.com/tasgon/observable/issues上报告错误",
+  "text.observable.entity_not_found": "在%s中找不到实体，他可能死了或者被移走了？",
+  "text.observable.get_field": "输入配置文件 URL/哈希...",
```

```suggestion
  "text.observable.get_field": "输入配置文件URL或哈希……",
```
请注意使用中文省略号
这里的profile是指什么